src/au/aulayer.cpp

```diff
@@ -110,18 +110,16 @@ CFURLRef aulayer::GetIconLocation ()
 
 void aulayer::InitializePlugin()
 {
-	if(!plugin_instance) 
-	{
-          AULOG::log( "SURGE:>> Constructing new plugin\n" );
-          AULOG::log( "     :>> BUILD %s on %s\n", __TIME__, __DATE__ );
-          //sub3_synth* synth = (sub3_synth*)_aligned_malloc(sizeof(sub3_synth),16);
-          //new(synth) sub3_synth(this);
-          
-          // FIXME: The VST uses a std::unique_ptr<> and we probably should here also
-          plugin_instance = new SurgeSynthesizer( this );
-          AULOG::log( "     :>> Plugin Created\n" );
+  if(!plugin_instance) 
```

Comment:
Sure! Just did that and pushed. Thanks! As long as @jsakkine doesn't now go and suggest I take it to 4. Chuckle.

aulayer.cpp and .h (the files) are a mess. I'll put in another issue to retab and remove cruft and so on.

Reply:
@whydoubt, Well, if we want to define any kind of conventions, it is an unnecessary evil to do this. It is better to do it now when the codebase is not yet otherwise mature.

Reply:
Cool. @whydoubt is better that we should keep the diffs minimal even if it means the tabs are gross. I actually made a tiny change to the pan2 PR to meet that goal since it had the same problem. Thanks folks!